### DISCUSSION TITLE
2 photos in 1

### DISCUSSION THREAD

**Discussion by MixFix228** [lllyasviel/Fooocus#3304] [Q&A] [answered]
Is there any way to merge 2 photos? like merge the face from one photo with the other? how do you do it?

**Comment by Ckrotky** [comment, marked as answer]
Use ☑️Input Image>☑️Image Prompt>☑️Advanced
Input in First slot you image with body and select PyraCanny (also set Stop At and Weight parameter)
Input in Second slot you image with Face and select FaceSwap (also set Stop At and Weight parameter)

**Reply by MixFix228** [reply]
thx u
